website/docs/reference/dbt-jinja-functions/config.md

```diff
@@ -1,7 +1,7 @@
 ---
 title: "config"
 id: "config"
-description: "The `config` variable manages user configuration for custom materialization."
+description: "Read this guide to understand the config Jinga function in dbt."
```

Comment:
`Jinga` should be `Jinja`

Reply:
https://deploy-preview-3261--docs-getdbt-com.netlify.app/reference/dbt-jinja-functions/config

content="Read this guide to understand the config Jinja function in dbt."